OrbitGl/Batcher.cpp

```diff
@@ -40,8 +42,8 @@ void Batcher::AddVerticalLine(Vec2 pos, float size, float z, const Color& color,
 void Batcher::AddLine(Vec2 from, Vec2 to, float z, const Color& color, const Color& picking_color,
                       std::unique_ptr<PickingUserData> user_data) {
   Line line;
-  line.start_point = Vec3(from[0], from[1], z);
-  line.end_point = Vec3(to[0], to[1], z);
+  line.start_point = Vec3(floorf(from[0]), floorf(from[1]), z);
```

Comment:
I would like to have some comment explaining why we are making this floorf.

Reply:
Good point. I've added a documentation for the Batcher-class to the header explaining that this is done and what the reasoning behind it is.